### DISCUSSION TITLE
commercial/technical-support

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#950] [Announcements]
# commercial/technical-support

技术支持 问题反馈 在使用项目的过程中，如果遇到问题，你可以先详细阅读本文档，未找到解决方案时，可以通过以下方式获取技术支持： 通过 Gitee Issues

https://docs.madong.tech/commercial/technical-support.html

**Comment by xiaodai1992625** [comment]
我想问下如果表名是  ma_1_news ma_1_news_data_0 这种表如何使用dao去执行model啊
